src/components/dataset/row.vue

```diff
@@ -46,6 +57,10 @@ export default {
 
 .dataset-row {
   .name { @include text-overflow-ellipsis; }
+  .entities {
+    text-align: right;
+    padding-right: 20px;
```

Comment:
In the screenshot in the release criteria, there looks to be more space between the "Total Entities" and "Latest Entity" columns. What do you think about increasing the padding? Does it help to make the padding a percentage?